homeassistant/components/togrill/event.py

```diff
@@ -0,0 +1,59 @@
+"""Support for event entities."""
+
+from __future__ import annotations
+
+from togrill_bluetooth.packets import Packet, PacketA5Notify
+
+from homeassistant.components.event import EventEntity
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.util import slugify
+
+from . import ToGrillConfigEntry
+from .const import CONF_PROBE_COUNT
+from .coordinator import ToGrillCoordinator
+from .entity import ToGrillEntity
+
+PARALLEL_UPDATES = 0
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ToGrillConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up event platform."""
+    async_add_entities(
+        ToGrillEventEntity(config_entry.runtime_data, probe_number=probe_number)
+        for probe_number in range(1, config_entry.data[CONF_PROBE_COUNT] + 1)
+    )
+
+
+class ToGrillEventEntity(ToGrillEntity, EventEntity):
+    """Representation of a Hue Event entity from a button resource."""
+
+    def __init__(self, coordinator: ToGrillCoordinator, probe_number: int) -> None:
+        """Initialize the entity."""
+        super().__init__(coordinator=coordinator)
+
+        self._attr_translation_key = "event"
+        self._attr_translation_placeholders = {"probe_number": f"{probe_number}"}
+        self._attr_unique_id = f"{coordinator.address}_event_{probe_number}"
```

Comment:
no need to add the platform in the unique id

Reply:
I'm not adding platform? I'm adding the base device.